module #3. REST API Advanced/web/src/main/java/com/epam/esm/controller/GiftCertificateController.java

```diff
@@ -62,4 +66,14 @@ public GiftCertificateController(GiftCertificateService giftCertificateService){
         giftCertificateService.delete(id);
         return ResponseEntity.noContent().build();
     }
+
+    private void addLinks(GiftCertificateDto certificateDto){
+        certificateDto.getTags().forEach(tagDto -> {
+            long tagId = Long.parseLong(tagDto.getId());
+            Link tagLink = WebMvcLinkBuilder.linkTo(WebMvcLinkBuilder.methodOn(TagController.class)
+                    .getById(tagId))
+                    .withRel("tag");
+            tagDto.add(tagLink);
```

Comment:
CertificateDto has all fields of every TagDto. Tag link direct to TagDto with all this fields. It's ok, or show only TagDto id in Certificate, and direct to full info by link?